components/collective-navbar/index.js

```diff
@@ -183,6 +185,50 @@ const CategoriesContainer = styled(Container)`
   }
 `;
 
+const accountPermissionsQuery = gqlV2/* GraphQL */ `
+  query AccountPermissions($slug: String!) {
+    account(slug: $slug) {
+      id
+      ... on Collective {
+        permissions {
+          id
+          addFunds {
+            allowed
+            reason
+          }
+        }
+      }
+      ... on Event {
+        permissions {
+          id
+          addFunds {
+            allowed
+            reason
+          }
+        }
+      }
+      ... on Project {
+        permissions {
+          id
+          addFunds {
+            allowed
+            reason
+          }
+        }
+      }
+      ... on Fund {
+        permissions {
+          id
+          addFunds {
+            allowed
+            reason
+          }
+        }
```

Comment:
Why not directly:

```suggestion
      permissions {
        id
        addFunds {
          allowed
          reason
        }
      }
```

Reply:
@Betree : This isn't possible I think as we don't have `permissions` on `Account`. In fact there's a ESlint warning if we do this; `ESLint: Cannot query field "permissions" on type "Account". Did you mean to use an inline fragment on "Bot", "Collective", "Event", "Fund", or "Host"?(graphql/template-strings)`

Reply:
The field was added to `AccountFields`, so the intent was clearly to add it to `Account`. Simply add the field definition in https://github.com/opencollective/opencollective-api/blob/d8551731c749832eb133928402709a95a5779846/server/graphql/v2/interface/Account.js#L61, it should solve the ESLint warning.

Reply:
Thanks, I missed this. Done. 👍🏽